_packages/native-preview/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "private": true,
+    "name": "@typescript/native-preview",
+    "version": "0.0.0",
+    "license": "Apache-2.0",
+    "author": "Microsoft Corp.",
+    "homepage": "https://www.typescriptlang.org/",
+    "description": "Preview CLI of the native TypeScript compiler port",
+    "keywords": [
+        "TypeScript",
+        "Microsoft",
+        "compiler",
+        "language",
+        "javascript",
+        "tsgo"
+    ],
+    "bugs": {
+        "url": "https://github.com/microsoft/typescript-go/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/typescript-go.git"
+    },
+    "type": "module",
+    "preferUnplugged": true,
+    "engines": {
+        "node": ">=20.6.0"
```

Comment:
What's the reasoning behind selecting this restriction here?

Reply:
For unflagged and sync `import.meta.resolve`, and Node 18 is EOL.